src/v2/connectors/repository/invoice-accounts.js

```diff
@@ -0,0 +1,45 @@
+const { InvoiceAccount } = require('../bookshelf');
+
+/**
+ * Find single InvoiceAccount with relations by ID
+ * @param {String} id
+ * @return {Promise<Object>}
+ */
+const findOne = async id => {
+  const result = await InvoiceAccount
+    .forge({ invoice_account_id: id })
+    .fetch({
+      withRelated: [
+        'company',
+        'invoiceAccountAddresses',
+        'invoiceAccountAddresses.address'
+      ],
+      require: false
+    });
+  return result ? result.toJSON() : null;
+};
+
+/**
+ * Find many InvoiceAccounts with relations by IDs
+ * @param {Array<String>} ids
+ * @return {Promise<Array>}
+ */
+const findWithCurrentAddress = async ids => {
+  const result = await InvoiceAccount
+    .collection()
+    .where('invoice_account_id', 'in', ids)
```

Comment:
camel case for invoice_account_id

Reply:
Camel-case doesn't seem to work here, presumably because Bookshelf drops down to the underlying Knex instance (and the camelcase is a Bookshelf plugin)